demos/react/ecommerce/src/Stats.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react'
+import { useInstantSearch } from 'react-instantsearch-hooks-web'
+
+export function Stats() {
+  const { results } = useInstantSearch()
```

Comment:
unless we specifically teach custom widgets, the basic Stats widget is probably sufficient right?